powermeter/slewrate.py

```
@@ -0,0 +1,71 @@
+
+    def wait_for_message(self, timeout=5):
+        """Pass through to wrapped powermeter."""
+        return self.wrapped_powermeter.wait_for_message(timeout)
+
+    def get_powermeter_watts(self) -> List[float]:
+        with self._lock:
+            raw_values = self.wrapped_powermeter.get_powermeter_watts()
+            current_time = time.time()
+    
+            if self.last_values is None or self.last_time is None:
+                # First reading – initialise without slewing
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)